src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperAttribute.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Internal.Web.Utils;
+
+namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
+{
+    public class TagHelperAttribute : IReadOnlyTagHelperAttribute
```

Comment:
Aren't you specifically comparing against other `IReadOnlyHelperTagAttributes` all the time? What functionality do you specifically lose here?